5/README.md

```diff
@@ -22,46 +28,76 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Tango Property Specification
 
- A Property is designed to represent any information of configuration in Tango.
+This specification is intended to formally document the Tango Property model.
+Runtime representation of Property in conforming Tango implementations MUST follow this specification.
 
 ### Goals
 
- The specification of Property aims to define a standard structure and type to use in any implementation of Tango.
+A Property represents a configuration parameter in Tango.
 
-Additionally, it aims to:
-
-* Provide the initial information when one starts a Tango Device
-
-* Be usable as keeping global variable in the Tango control system
-
-* Provide meta data complementing any data in Tango
+There are 4 kinds of Tango Properties:
 
+ *  Class Property: a Property related to a Device Class (RFC-2).
+ *  Device Property: a Property related to a Device (RFC-2).
+ *  Attribute Property: a Property related to an Attribute (RFC-4).
+ *  Free Property: a Property defined for the entire Control System (RFC-1) i.e not related to a specific Class, Device or Attribute.
 
 ### Use Cases
 
-There are many use cases for the usage of a Property:
-
-* To connect a real network equipment a Property can define the target IP address
-
-* To change the representation of an information in order to be more user friendly a Property can define a new data format for the graphical user interface to convert the raw data.
+Here are some Tango Property use cases:
 
+* To connect a real network equipment a Property can define the target IP address or hostname
+* To change the representation of an information in order to be more user friendly a Property can define a new data format for the graphical user interface to convert the raw data
+* To configure some alarms on Attribute Value (See RFC-4)
+* To store the Attribute value of a Memorized Attribute (See RFC-4)
+* To configure the labels associated to a Tango:DevEnum attribute values
+* ...
 
 ## Specification
 
-A Tango Property is a strict definition of a pair of key/value
-* configuration are represented in the form of properties.
-* data are represented in the form of attributes.
-* actions are represented in the form of commands.
-
-Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, Attribute, Command following the rules below:
-
-
-A Tango Property is a strict definition of a pair of key/value
-* The Property SHALL have one key, called Property Name
-* The Property SHALL have one value
+A Tango Property is a strict definition of a pair of key/value:
+  * The Property SHALL have one key, called Property Name
+  * The Property SHALL have one value, which could be empty, called Property Value
+
+A Tango Property MAY be persisted in the Tango Database (See RFC-1 and RFC-6) or into a file if no Tango Database is used.
```

Comment:
As I wrote to Lorenzo, a default property value can be defined in POGO and if this default value is good for a particular device, the property does not need to be persisted in the DB for this specific use case. So there are cases where a property does not need to be persisted... This is why I've written "MAY be persisted".
It is confusing it seems. Any suggestion to improve this?

Reply:
It is sort of borderline case... So, maybe it would be useful to describe the mechanism a bit in the rfc, even if it is not exactly specification...

Reply:
> It is sort of borderline case... So, maybe it would be useful to describe the mechanism a bit in the rfc, I mean the default value in POGO/source code, even if it is not exactly specification...

Well, there is already this sentence which can probably be improved:

> Device Properties and Class Properties MAY have a _Default Value_ which can be used if the Property Value has not been persisted.

Reply:
Yes. But it is further on in the spec. What about moving the sentence "A Tango Property MAY be persisted in..." after this one? Moreover, if the Property value is not persisted, wouldn't the default value be used... by default? Propose to write the two sentences as:
- A Tango Property MAY be persisted in the Tango Database (See RFC-1 and RFC-6) or into a file if no Tango Database is used.
- Device Properties and Class Properties MAY have a Default Value which _is_ used if the Property Value has not been persisted.
(emphas_is_ just for this comment)

Reply:
> Yes. But it is further on in the spec. What about moving the sentence "A Tango Property MAY be persisted in..." after this one?

It was in this order because I thought it was better to have mentioned the possibility for a property to be persisted before.

> Moreover, if the Property value is not persisted, wouldn't the default value be used... by default? Propose to write the two sentences as:
> 
>     * A Tango Property MAY be persisted in the Tango Database (See RFC-1 and RFC-6) or into a file if no Tango Database is used.
> 
>     * Device Properties and Class Properties MAY have a Default Value which _is_ used if the Property Value has not been persisted.
>       (emphas_is_ just for this comment)

Your proposal sounds good to me! The 2 sentences will be regrouped together.